### DISCUSSION TITLE
RP2 Pico2-W BLE Library

### DISCUSSION THREAD

**Discussion by snphysic** [micropython/micropython#16367] [Libraries & Drivers] [answered]
I recently purchased an RP2 Pico 2 W for my project, intending to use its Bluetooth functionality. Unfortunately, I’ve encountered an issue:

The Pico W firmware does not work with the Pico 2 W.
The Pico 2 firmware, while functional, does not support Bluetooth.
Could you kindly let me know if there are plans to release a firmware version for the Pico 2 W that supports Bluetooth? I am eager to use this feature and would appreciate any updates on the timeline.

Thank you for your time and assistance.

**Comment by scruss** [comment, marked as answer]
duplicate of this: [Pico 2 W Firmware · micropython · Discussion #16363](https://github.com/orgs/micropython/discussions/16363)
which has an answer
